### ISSUE TITLE
VPAT regarding usability by individuals with disabilities.

### ISSUE THREAD

**Issue by JoltRabbit** [wix/Detox#1874]
I am attempting to get your product approved for usage within a government agency that I am working with and I was hoping you could provide me a VPAT to confirm that it meets the standards set in section 508 of the Rehabilitation Act of 1973.  This document will basically describe if it is usable by individuals with disabilities.

Could you please provide me with a VPAT for Detox?  Here is more information on what VPAT is: [https://www.section508.gov/sell/vpat](https://www.section508.gov/sell/vpat)

Thank you!
Tom

**Comment by LeoNatan** [comment]
Hello,

Can you please elaborate what a VPAT is?

Thanks

**Comment by JoltRabbit** [comment]
Hello @LeoNatan,

"A Voluntary Product Accessibility Template (VPAT™) is a document that explains how information and communication technology (ICT) products such as software, hardware, electronic content, and support documentation meet (conform to) the Revised 508 Standards for IT accessibility. VPATs™ help Federal agency contracting officials and government buyers to assess ICT for accessibility when doing market research and evaluating proposals." - This paragraph was quoted from the site: https://www.section508.gov/sell/vpat

In summary, it is a documented voluntary review of any software of the accessibility of the software for people with disabilities.  If you have a VPAT it makes things a lot easier to get it approved for usage with government agencies, such as the one I am working with now.

If there is not a VPAT for detox and you know that could you let me know?  No could be the answer at the current time, I just need to know.

Thank you!

**Comment by LeoNatan** [comment]
Thanks for your reply. 
There isn’t a VPAT document for Detox at this time, but I can push it. 

Since Detox is not a traditional application or software—it has no UI of its own, for example—would that mean that creating the document for Detox is a bureaucratic action only? I ask just to make sure there isn’t a misunderstanding, not because we won’t do it.

**Comment by JoltRabbit** [comment]
Thank you @LeoNatan!  I really appreciate your responses!

I would say it's a little of both.  To get software approved it needs to go through a review.  The review will give precedence to options with VPATs.  It doesn't guarantee to dismiss options, with or without but gives a great advantage over the others.  So there's the bureaucratic side.

On the other side, the overall intent of the VPAT and it's inclusion and prioritization of software that voluntarily includes people with disabilities is something I could stand behind.

And to close, I do understand it doesn't have a UI so not really directly applicable (AFAIK).  Which would probably make getting the VPAT a simple thing; I would think?  And you could put the VPAT on your site for the public to see to instantly get Detox in the running as a prime candidate for government evaluations, which wouldn't hurt at all.

Your responses are very much appreciated!  I have made my submission today, so no urgency would be needed on my account.  As an FYI, none of the products I have reviewed and proposed currently have VPATs but it's something I would encourage to give the product that edge with government usage and advocate support for the cause.  Sorry if I sound preachy, I just think it's a cool thing.

Have a great day and Thank you!
Tom

**Comment by LeoNatan** [comment]
Thanks for the explanation! We’re looking into it.

**Comment by LeoNatan** [comment]
@JoltRabbit 

Hello,
Could you please comment on which standard specifically you need? Is it the 508?

Thanks

**Comment by LeoNatan** [comment]
Hello,

VPAT document was uploaded here: https://github.com/wix/Detox/blob/master/docs/Detox%20VPAT2.3Rev508.pdf

Thanks
